SecurityPkg/SecurityPkg.dec

```diff
@@ -52,16 +52,16 @@
   #
   Tpm2DeviceLib|Include/Library/Tpm2DeviceLib.h
 
-  ## MU_CHANGE
-  ##  @libraryclass  Oem library hooks for additional TPM 2.0 initialization
-  #
-  OemTpm2InitLib|Include/Library/OemTpm2InitLib.h
-
-  ##  @libraryclass  This library contains helper functions to perform a detailed debugging of
-  ##  TPM transactions
-  #
-  Tpm2DebugLib|Include/Library/Tpm2DebugLib.h
-
+  ## MU_CHANGE
+  ##  @libraryclass  Oem library hooks for additional TPM 2.0 initialization
+  #
+  OemTpm2InitLib|Include/Library/OemTpm2InitLib.h
+
+  ##  @libraryclass  This library contains helper functions to perform a detailed debugging of
+  ##  TPM transactions
+  #
+  Tpm2DebugLib|Include/Library/Tpm2DebugLib.h
+

```

Comment:
Looks like this change isn't needed?

Reply:
sorry, why is this not needed? Do you mean this entry in the DEC or the commit as a whole?